frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -1,10 +1,80 @@
 import { kea } from 'kea'
+import api from 'lib/api'
+import { errorToast, toParams } from 'lib/utils'
+import { cleanFunnelParams, funnelLogic } from 'scenes/funnels/funnelLogic'
+import { ViewType } from 'scenes/insights/insightLogic'
+import { cohortLogic } from 'scenes/persons/cohortLogic'
+import { ActionFilter, PersonType } from '~/types'
 import { personsModalLogicType } from './personsModalLogicType'
+import { parsePeopleParams, TrendPeople, trendsLogic } from './trendsLogic'
 
 export const personsModalLogic = kea<personsModalLogicType>({
+    connect: {
+        values: [trendsLogic, ['filters']],
+    },
```

Comment:
`trendsLogic` has a key on the `dashboardId` prop, and thus if you connect to it here, it'll connect to the default `all_trends` key. This will cause issues later with clicking on datapoints in dashboards.

You should either find a way to pass the `dashboardId` prop also into personsModalLogic, so that you could pass it to the right `trendsLogic` to get the right `filters` (and below in the file other values)... or, decouple personsModalLogic completely from trendsLogic. These filters can probably be passed in through actions? 

I didn't dig deep enough to know what would be the better option. If you'd like me to look further or get stuck, let me know :).

Reply:
Passing the filters through actions seems to work! I've clicked around and it's opening persons modal just fine.

I think there's something weird happening on my trends persons point though where I'm not getting results back, but I checked it out on master and the same issue is happening there too, so I don't _think_ this is causing that but I'll double check just to be sure

Reply:
nvm, i'm dumb, this is because of that same existing clickhouse persons bug, lol I keep forgetting. But this is working, I was able to check locally for lifecycles + funnels, and on production I just copied and pasted the endpoint from local and it also worked for the other insights 😄